camayoc/config.py

```diff
@@ -54,9 +76,32 @@ def get_settings_files(xdg_config_dir, xdg_config_file):
     return settings_files
 
 
-_CONFIG = Dynaconf(settings_files=get_settings_files("camayoc", "config.yaml"))
+def get_settings(path=None) -> Configuration:
+    if not path:
+        settings_files = get_settings_files("camayoc", "config.yaml")
+        dynaconf_validators = default_dynaconf_validators
+    else:
+        settings_files = [path]
+        dynaconf_validators = []
+
+    raw_settings = Dynaconf(
+        settings_files=settings_files,
+        validators=dynaconf_validators,
+    )
+
+    to_validate_settings = {key.lower(): value for key, value in raw_settings.as_dict().items()}
```

Comment:
That is addressed in 68bfddc959e80bb1342de1815e10f876a7642cf1 . The reason for this error was that dynaconf validators were setting `openshift` as a dict, but you were already expecting a list.

I am using dynaconf validators to set default values of some mandatory keys - this way we can get pydantic object even if there is no configuration file present.

Now, if you try to use a setting where openshift is a dict (instead of list of dicts), you will get:
```
openshift
  value is not a valid list (type=type_error.list)
```
Which I think is pretty good error message.